plugins/dependencies/src/main/kotlin/dependencies/Square.kt

```diff
@@ -7,10 +7,51 @@ import org.gradle.kotlin.dsl.IsNotADependency
 @Incubating
 object Square {
 
+    /**
+     * OkHttp - Square's meticulous HTTP client for Java and Kotlin
+     * - [Website and documentation](https://square.github.io/okhttp/)
+     * - [GitHub](https://github.com/square/okhttp)
+     * - [CHANGELOG](https://github.com/square/okhttp/blob/master/CHANGELOG.md)
+     *
+     */
     val okHttp3 = OkHttp3
+
+    /**
+     * Retrofit - A type-safe HTTP client for Android and the JVM
+     *
+     * - [Website](https://square.github.io/retrofit/)
+     * - [GitHub square/retrofit](https://github.com/square/retrofit)
+     * - [CHANGELOG](https://github.com/square/retrofit/blob/master/CHANGELOG.md)
+     * - [Consuming APIs with Retrofit | CodePath Android Cliffnotes](https://guides.codepath.com/android/Consuming-APIs-with-Retrofit#references)
```

Comment:
We don't control how outdated a third-party article can become, yet, we are helping our users update (diverging further from these possibly outdated articles at a quicker rate).

Also, we might never find the best article on a library, so let's stick to the official and blog posts from authors and major contributors, and for the rest, or when official content is lacking, let the users decide what to look for as a reference based on the metrics they value like time of publication, programming language (Kotlin vs Java), platform (Android, JVM, other…), search engines and critical sense do a great job for that.